src/Dommel/DommelMapper.cs

```diff
@@ -2048,7 +2048,7 @@ private sealed class MySqlSqlBuilder : ISqlBuilder
         {
             public string BuildInsert(string tableName, string[] columnNames, string[] paramNames, PropertyInfo keyProperty)
             {
-                return $"insert into {tableName} ({string.Join(", ", columnNames)}) values ({string.Join(", ", paramNames)}); select LAST_INSERT_ID() id";
+                return $"insert into {tableName} ({"`" + string.Join("`, `", columnNames) + "`"}) values ({string.Join(", ", paramNames)}); select LAST_INSERT_ID() id";

```

Comment:
A string starting with `$` means its an _interpolated_ string. You can use C# code in the string inside the `{` and `}` brackets instead of concatenating strings using `+`:

```cs
return $"insert into {tableName} (`{string.Join("`, `", columnNames)}`) values ({string.Join(", ", paramNames)}); select LAST_INSERT_ID() id";
```